habitat/datasets/utils.py

```diff
@@ -85,6 +85,15 @@ def _build(self):
     def idx2word(self, n_w):
         return self.word_list[n_w]
 
+    def token_idx_2_string(self, tokens):
```

Comment:
```suggestion
    def token_idx_2_string(self, tokens: Iterable[int] ) -> str: 
```
You will need to add the corresponding import of Iterable from typing too.